discovery-provider/src/models.py

```diff
@@ -156,6 +156,7 @@ class User(Base):
     primary_id = Column(Integer, nullable=True)
     secondary_ids = Column(postgresql.ARRAY(Integer), nullable=True)
     replica_set_update_signer = Column(String, nullable=True)
+    has_collectibles = Column(Boolean, nullable=False, default=False, server_default='false')
```

Comment:
does the default of False do a backfill for every user? Personally I'd err on the side of making all new fields nullable=True to avoid any unexpected behavior here and immediately after the migration the majority of users _will_ have this field as null

Reply:
unless i've misunderstood

Reply:
is this what server_default is intended to handle?

Reply:
it will be backfilled for every user, yep!
having this cleanly true / false saves a lot of headache on the consumer side because you have 2 possibilities instead of 3

Reply:
ok great